extensions/wikia/Chat2/NodeApiClient.class.php

```diff
@@ -110,7 +127,9 @@ static private function makeRequest($params){
 		global $wgReadOnly;
 		wfProfileIn( __METHOD__ );
 		$response = "";
-		
+
+		ChatHelper::debug( __METHOD__ . ': Method called ', $params );
```

Comment:
I don't have a problem including a few extra lines to capture this.  The difference between debug and info is fairly subjective and I could see switching other logging calls in this PR to debug in the future if it turns out they produce a lot of output that is not always needed.